scripts/tank_cmd.py

```diff
@@ -1104,7 +1104,10 @@ def run_engine_cmd(log, pipeline_config_root, context_items, command, using_cwd,
         console_logout()
         sys.exit()
```

Comment:
yes sorry, i meant the other one! But is it worth doing `sys.exit(0)` or is that overkill?